webportal/src/app/job/job-view/job-view.component.js

```diff
@@ -87,7 +87,7 @@ const loadJobs = () => {
     success: (data) => {
       loading.hideLoading();
       if (data.error) {
-        alert(data.message);
```

Comment:
>alert( [](start = 8, length = 6)

why comment it?

Reply:
Debug use... Because the page keep popping out annoying 'page not found' alerts when I debug locally. Reverted.

Reply:
By the way @fanyangCS : How may I add a group of people (the team) into the reviewer list?

Reply:
don't know 

---
In reply to: [158664983](https://github.com/Microsoft/pai/pull/136#discussion_r158664983) [](ancestors = 158664983)